src/dblib/dblib.c

```diff
@@ -725,6 +725,7 @@ dblogin(void)
 	/* set default values for loginrec */
 	if (!tds_set_library(loginrec->tds_login, "DB-Library")) {
 		dbperror(NULL, SYBEMEM, errno);
+		free(loginrec->tds_login);
```

Comment:
I think `tds_free_login` would be better here, more symmetrical.

Reply:
thanks, fixed